### PR TITLE
Adding SQL2014

### PR DESCRIPTION
**Pull request by get-itips**
Closes https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/issues/2555

@tseward can you kindly review? Thanks

### CODE REVIEW

**Review by tseward — APPROVED** [review]

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointServer/install/hardware-and-software-requirements-0.md [hunk 1]
```diff
@@ -133,6 +133,8 @@ Minimum requirements for a database server in a farm:
   
 - One of the following:
     
+  - The 64-bit edition of SQL Server 2014 (requires May 2014 Cumulative update for SharePoint Server 2013).
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointServer/install/hardware-and-software-requirements-0.md:136]:
```suggestion
  - The 64-bit edition of SQL Server 2014 (requires May 2014 cumulative update for SharePoint Server 2013).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@SerdarSoysal PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**